db/db.go

```diff
@@ -0,0 +1,157 @@
+package db
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"sync"
+
+	"github.com/mikloslorinczi/infra-exec/common"
+	"github.com/pkg/errors"
+	"github.com/rs/xid"
+)
+
+// I is the interface of the JSON DB.
+type I interface {
```

Comment:
Name `I` does not obey to the go naming conventions.

https://golang.org/doc/effective_go.html#interface-names